internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -107,27 +108,48 @@ func (h *HighPerformanceHooks) PreStop(ctx context.Context, c *oci.Container, s
 			return errors.Wrap(err, "set CPU load balancing")
 		}
 	}
+
 	// enable the IRQ smp balancing for the container CPUs
 	if shouldIRQLoadBalancingBeDisabled(s.Annotations()) {
 		if err := setIRQLoadBalancing(c, true, irqSmpAffinityProcFile); err != nil {
 			return errors.Wrap(err, "set IRQ load balancing")
 		}
 	}
+
 	// no need to reverse the cgroup CPU CFS quota setting as the pod cgroup will be deleted anyway
 
 	return nil
 }
 
+func annotationValueDeprecationWarning(annotation string) string {
+	return fmt.Sprintf("The usage of the annotation %q with value %q will be deprecated under 1.21", annotation, "true")
+}
+
```

Comment:
can we have this function underneath the last invocation of it? it improves readability imo

Reply:
sure will move it

Reply:
done